.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Thank you for contributing to `too-many-hooks`! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we will be best able to review your PR.
+-->
+
+### Why:
+
+Closes ISSUE
+
+<!-- If there's an existing issue for your change, please replace ISSUE above with a link to the issue.
+If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/yobgob/too-many-hooks/issues/new. -->
+
+### What's being changed (if available, include any code snippets, screenshots, or gifs):
+
+<!-- Let us know what you are changing. Share anything that could provide the most context.
+If you have made changes to stories please include screenshots of the changed / added stories-->
```

Comment:
Shouldn't chromatic take care of the diff for us?

Reply:
As it is now because of limited permissions from forks and trying to keep the Chromatic token secret, Chromatic runs after a PR merges into `main` or if one of us with the Chromatic secret checks out the branch and runs it manually. This is good to make sure the changes are what we expected but does not necessarily provide help in the review process so screenshots are still valuable.